modules/planning/lattice/trajectory_generation/trajectory1d_generator.cc

```diff
@@ -28,14 +28,43 @@
 #include "modules/common/log.h"
 #include "modules/planning/common/planning_gflags.h"
 #include "modules/planning/lattice/trajectory1d/constant_deceleration_trajectory1d.h"
-#include "modules/planning/lattice/trajectory1d/standing_still_trajectory1d.h"
 #include "modules/planning/lattice/trajectory1d/lattice_trajectory1d.h"
+#include "modules/planning/lattice/trajectory1d/standing_still_trajectory1d.h"
 #include "modules/planning/math/curve1d/quartic_polynomial_curve1d.h"
-#include "modules/planning/math/curve1d/quintic_polynomial_curve1d.h"
 
 namespace apollo {
 namespace planning {
 
+namespace {
+
+// A common function for trajectory bundles generation with 
+// a given initial and  end conditions.
+typedef std::array<double, 3> Init_Condition_T;
+typedef std::vector<std::pair<std::array<double, 3>, double>> End_Conditions_T;
+void GenerateTrajectoryBundlesWithCondition(
```

Comment:
Maybe you can move the quintic polynomial related code to a common function, because they are majority.

Reply:
The common function uses different initial and end conditions to generate curve bundles. So it can be used here. Please check the following line in the common function  carefully.  
```
auto ptr_trajectory1d = std::make_shared<LatticeTrajectory1d>(
        std::shared_ptr<Curve1d>(new QuarticPolynomialCurve1d(
            init_condition, end_state, end_condition.second)));
```

Reply:
OK. I'll modify it and submit again. Thank you.